.github/workflows/central-publish.yml

```diff
@@ -25,6 +25,9 @@ jobs:
                     distribution: 'temurin'
                     java-version: 17.0.7
 
+            -   name: Grant execute permission for gradlew
+                run: chmod +x gradlew
+
```

Comment:
Do we really need this step? Usually the GitHub runners has the execution rights for Gradle. Shall we also update the checkout actions to v3?

Reply:
Fixed in [e1c6759](https://github.com/xlibb/module-pdfbox/pull/9/commits/e1c67595c6c2590595617d6e485a699b73b55d38)